cmd/spire-server/cli/entry/create.go

```diff
@@ -42,9 +42,12 @@ type CreateConfig struct {
 
 	// Whether or not the entry represents a node or group of nodes
 	Node bool
+
+	// Expiry of entry
+	Expiry int64
```

Comment:
I think this would be better named `EntryExpiry` or similar. There has been some confusion historically around what exactly `TTL` means, and I worry that `Expiry` will only make it worse. Something that more clearly communicates that this is the expiry for the entry itself would be great.

This is most important when considering the points where the option is exposed e.g. registration api and cli utilities... If you want to leave the field on the underlying model and/or the datastore proto as `Expiry` I think that's fine for now.